lib/kernel-manager.js

```diff
@@ -86,11 +86,26 @@ class KernelManager {
 
   startKernel(kernelSpec, grammar, onStarted) {
     const displayName = kernelSpec.display_name;
+    let kernelStartDir;
+    let currentPath = getEditorDirectory(store.editor);
 
     log("KernelManager: startKernel:", displayName);
 
+    switch (atom.config.get("Hydrogen.startDir")) {
+      case "firstProjectDir":
+        if (atom.project.getPaths().length > 0) {
```

Comment:
If this condition isn't fulfilled it falls back to `case "projectDirOfFile":` which will return `/`. I think it should fall back to `currentPath` instead.